cmd/get_projects.go

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2018 github.com/devopsctl authors
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+var getProjectsCmd = &cobra.Command{
```

Comment:
This command has a lot of flags with different output. Can you provide at least 3 examples using [cobra.Command.Examples](https://godoc.org/github.com/spf13/cobra#Command) string?

```go
Examples: `
# display projects only with issues enabled
gitlabctl get projects --with-issues-enabled

# simple view
gitlactl get projects --simple
`
```

Reply:
Done